site/content/en/docs/tasks/run_flux.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Run A Flux Minicluster"
+date: 2022-02-14
+weight: 6
+description: >
+  Run a Kueue scheduled Flux minicluster.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Flux Operator's](https://flux-framework.org/flux-operator/) Mini Clusters.
+
+For more information on Flux Operator visit [Flux Operator](https://flux-framework.org/flux-operator/).
+
+
+## Before you begin
+
+Please check [Run A Job](/docs/tasks/run_jobs) for details on the initial cluster setup and queue configuration.
+
+## 1. MiniCluster definition
+
+Since a Flux MiniCluster uses  a `batch/job` to run the actual workload, no extra components are needed for it to be managed by Kueue, However the following aspects should be taken into account:
+
+### a. Queue selection
+
+The target local queue should be specified in the `spec.jobLabels` section of the minicluster configuration.
+
+```yaml
+  jobLabels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Configure the resource needs
+
+The resource needs of the workload can be configured in the `spec.container[].resources` sections of the minicluster configuration.
+
+```yaml
+spec:
+  containers:
+    - image: <image>
+      resources:
+        requests:
+          cpu: 4
+          memory: "200Mi"
+```
+
+## 2. Configuration example
+
+```yaml
+apiVersion: flux-framework.org/v1alpha1
+kind: MiniCluster
+metadata:
+  generateName: flux-sample-kueue-
+spec:
+  size: 1
+  containers:
+    - image: ghcr.io/flux-framework/flux-restful-api:latest
+      command: echo hello world 
```

Comment:
Is this a meaningful command @vsoch?

Is there a hello HPC? calculate pi, etc?

Reply:
Oh yeah you mean:

```bash
echo hello hpc :wave: 
```
:laughing: 

We have quite a few examples https://github.com/flux-framework/flux-operator/tree/main/examples and actually the one I'd recommend (with a pre-built container that does something marginally interesting that uses MPI) is... LAMMPS!

https://github.com/flux-framework/flux-operator/blob/main/examples/tests/lammps/minicluster.yaml

We do have a hello-world example https://github.com/flux-framework/flux-operator/blob/main/examples/tests/hello-world/minicluster.yaml but I'd stay away from it, because it runs so quickly you'd probably miss it.

Reply:
switch to `sleep 10`, to simulate some work